woocommerce/woocommerce-ios
# Configure subheadline button title color in `WordPressAuthenticatorStyles`

Fixes #151. 

**Background:** In WCiOS (host app) all UIButton instances had their UIAppearance set with a text color of `wooPurple`. Setting UIAppearance for all UIButton instances causes bugs because it's too general. For example, the Logout button should be a dark red "destructive action" color, but it's purple. 

## To Test

1. check out this branch
2. check out this `WPAuth` branch: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/16 
3. build / run `WPAuth` framework
4. delete any existing WCiOS app builds
5. open WCiOS podfile
6. point to the WordPressAuthenticator development pod (`pod 'WordPressAuthenticator', :path => '/your/path/to/WPAuth'`
7. `pod install`
8. run WCiOS
9. verify the subheadline buttons remain purple (see list of button titles below)
10. attempt to paste some text into the password field the "Paste" tooltip should have a white text color instead of purple.
11. after logging in, an "Unable to find WooCommerce stores connected to this account" message will appear. Tap the "Continue" button. 
12. navigate to Orders > select an order > check the "Fulfill order" button. The button text color should be white.

### Subheadline buttons found in WPAuth
(Not all of these buttons appear in WCiOS.)
- [ ] Log in with Google
- [ ] Log in by entering your site address
- [ ] Enter your password instead
- [ ] Forgot password
- [ ] Lost your password
- [ ] Send another code
- [ ] Need help finding your site address

#### Title color issues checklist
- [ ] Paste tooltip is white text not purple 
<img width="69" alt="screen shot 2018-07-11 at 12 23 20 pm" src="https://user-images.githubusercontent.com/1062444/42700672-0fb812ee-868a-11e8-9af2-2b1472887983.png">

- [ ] Fulfill order button has white text not purple 
<img width="240" alt="screen shot 2018-07-13 at 10 47 22 am" src="https://user-images.githubusercontent.com/1062444/42700693-24cadbc6-868a-11e8-82c9-654ae37d3072.png">
